packages/wdio-browserstack-service/src/Percy/PercyBinary.ts

```diff
@@ -1,17 +1,15 @@
 import url from 'node:url'
 import yauzl from 'yauzl'
-
-import { createRequire } from 'node:module'
-const require = createRequire(import.meta.url)
-
-const fs = require('node:fs')
+import fs from 'node:fs'
 import fsp from 'node:fs/promises'
-const { https } = require('follow-redirects')
+import pkg from 'follow-redirects'
+const { https } = pkg
```

Comment:
Can't we use directly? 
```suggestion
import { https } from 'follow-redirects`
```

Reply:
Nope, on trying that we get following error, hence changed it to the recommended method :-

import { https } from 'follow-redirects';
         ^^^^^
SyntaxError: Named export 'https' not found. The requested module 'follow-redirects' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'follow-redirects';
const { https } = pkg;